src/main/java/io/phdata/streamliner/schemadefiner/mapper/SnowflakeMapper.java

```diff
@@ -96,25 +123,37 @@ private static SnowflakeTable mapSchemaCrawlerTable(Table table, Snowflake desti
 
     return new SnowflakeTable(
         table.getName(),
-        getSnowflakeTableName(table.getName(), destination),
+        getSnowflakeTableName(table.getName()),
```

Comment:
This is a Hive table right, why is it named 'getSnowflakeTable`?

Reply:
This is for the target. Our destination is Snowflake. This method gets the target Snowflake table name based on the table naming strategy applied. That's why it is named `getSnowflakeTableName`

Reply:
I see

Reply:
I see